docs/src/main/asciidoc/doc-contribute-docs-howto.adoc

```diff
@@ -205,16 +280,16 @@ Each IDE integration has its own configuration requirements. The Visual Studio C
 Submit your proposed changes to the core Quarkus docs by {gh-pull-requests-fork}[creating a pull request] against the `main` branch of the Quarkus repository from your own {gh-about-forks}[repository fork].
 
 Reviews for code and documentation have different (but overlapping) participants.
-To simplify collaborative review, either isolate changes to docs in their own PRs,
+To simplify collaborative review, either isolate changes to docs in their PRs
 or ensure that the PR has a single, focused purpose. For example:
 
 - Create a single PR that adds a configuration option for an extension and updates related materials (how-to, reference) to explain the change.
-- Create a single PR for related changes to a group of documents, some examples:
-correcting the usage of a term, correcting a recurring error, or moving common content into a shared file.
+- Create a single PR for related changes to a group of documents; some examples:
+Correcting the usage of a term, correcting a recurring error, or moving common content into a shared file.
 - If there are extensive code changes and documentation changes,
 create a separate PR for the documentation changes and include the relationship in the issue description.
 
```

Comment:
Do we want to add a link to https://github.com/quarkusio/quarkus/blob/main/COMMITTERS.adoc here or in the references section?

Reply:
Updated. Thanks.